### DISCUSSION TITLE
Issue with Floating Terminal in Snacks.terminal.open

### DISCUSSION THREAD

**Discussion by mxCynic** [folke/snacks.nvim#155] [General]
In the terminal.md, I saw the statement  _"if cmd is provided, the window will be opened in a floating window"._ So I try to use 
```
:lua Snacks.terminal.open("ls", opts)
```
It just run this cmd in a floating window and shutdown the windows in flash.If I use "python" instead "ls", the floating windows will wait me.  
My goal is to have a floating terminal that behaves like the default terminal in Lazyvim—staying open after executing the command. How can I achieve this?

https://github.com/user-attachments/assets/7ee3085b-888e-4a30-8d20-a69bb4195a1b

**Comment by folke** [comment]
Add `interactive = false` for this

**Reply by mxCynic** [reply]
I'm sorry I didn't make myself clear earlier. My ultimate goal is to make the terminal opened with the default shortcut key float by default. Previously, when I used toggleterm.nvim, I could set 
```
direction = 'float'
``` 
to make newly opened terminals float.  
 How can I achieve the same with  snacks.nvim terminal?

**Reply by folke** [reply]
Also add position=float
And please read the docs. It has all that info

**Reply by mxCynic** [reply]
Tanks, I finish it.
I have read the docs again and find the "position" at  *snacks-win-config*, but I only read terminal.md at github previously, that's why I didn't  find the answer. I'm sorry for my careless.
